src/wavesurfer.js

```diff
@@ -441,6 +441,39 @@ var WaveSurfer = {
     },
 
     /**
+     * Unmounts the wavesurfer instance from the page, but keeps the audio
+     * player in memory
+     */
+    unmount: function (selector) {
+        if (this.container.children.length === 0) {
+            var wsNode = document.querySelectorAll(selector)[0];
+            var newNode = wsNode.cloneNode();
+            this.container.appendChild(newNode.children[0]);
```

Comment:
I don't quite get this logic with the clone and append. If you want to remove the element, wouldn't simply calling `this.container.removeChild(this.drawer.wrapper)` be enough?